cloudformation/sorry-cypress.yml

```diff
@@ -313,7 +313,6 @@ Resources:
     Type: AWS::S3::Bucket
     Properties:
       BucketName: !Join ['-', [!Ref 'AWS::StackName', 'test-results']]
-      AccessControl: PublicRead
```

Comment:
This resolves a breaking change due to AWS updating allowed default settings on new buckets.

Reply:
Sorry for my ignorance, what do you mean here?